Completed/ModelExplorer/Assets/Scripts/Interactible.cs

```diff
@@ -14,7 +14,7 @@ public class Interactible : MonoBehaviour
 
     void Start()
     {
-        defaultMaterials = GetComponent<Renderer>().materials;
+        defaultMaterials = GetComponent<Renderer>()?.materials;
```

Comment:
It did build for me, but you are correct. null propagating is a C# 6 feature. Ill step through this process again tonight and see what I may have done wrong.  It did build for me (which is weird), and adding the operator did seem to fix the null issue I was receiving.